framework/src/actions/MaterialOutputAction.C

```diff
@@ -262,14 +282,19 @@ MaterialOutputAction::getParams(const std::string & type,
                                 const std::string & variable_name,
                                 const MaterialBase & material)
 {
+  // Use a suffix to disambiguate variables and material properties with the same name
+  const auto output_suffix = material.isParamValid("material_output_suffix")
```

Comment:
Any reason you don't just default this param to `""`?

Reply:
not really. I changed it to be default '' now